### PR TITLE
CSS "contain-intrinsic-size" property

### PR DESCRIPTION
**Pull request by ramiy**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any

New CSS property.

**Required for:**

https://developer.mozilla.org/en-US/docs/Web/CSS/contain-intrinsic-size

**Resources:**

https://web.dev/content-visibility/

### CODE REVIEW

**Comment by rachelandrew** [comment]
@ramiy I believe this went into Chrome 83, see: https://www.chromestatus.com/feature/5737051062272000

File: css/properties/contain-intrinsic-size.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```
Comment (ddbeck) [inline css/properties/contain-intrinsic-size.json:27]:
```suggestion
              "version_added": "69"
```

File: css/properties/contain-intrinsic-size.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            },
+            "opera_android": {
+              "version_added": false
```
Comment (ddbeck) [inline css/properties/contain-intrinsic-size.json:30]:
```suggestion
              "version_added": "59"
```

File: css/properties/contain-intrinsic-size.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
```
Comment (ddbeck) [inline css/properties/contain-intrinsic-size.json:39]:
```suggestion
              "version_added": "13.0"
```

File: css/properties/contain-intrinsic-size.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            },
+            "webview_android": {
+              "version_added": false
```
Comment (ddbeck) [inline css/properties/contain-intrinsic-size.json:42]:
```suggestion
              "version_added": "83"
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @ramiy. A couple suggestions to cascade the Chrome data to the other Chromium-derived browsers. Otherwise, looking good.

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Pull request merged**